src/components/ContentOpenWith/ContentOpenWith.scss

```diff
@@ -1,6 +1,8 @@
 @import '../variables';
 
 .be.bcow {
+    font-family: Lato, "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
This is necessary because the fonts are not a part of the component, but assumed to be on the page. Adding this here will also transfer to the new tab. I'll look into the variable.